src/main/java/com/omega/Proyecto/omega/Service/ServiceTouristicServ.java

```diff
@@ -35,20 +35,33 @@ public boolean deleteService(Long id) throws ObjectNotFoundException {
         return true;
     }
 
+    //Cuando haga el merge con la rama de implementacion de errores
+    //modifico este metodo para que busque  solo id's activos.
```

Comment:
```suggestion
    //Cuando haga el merge con la rama de implementacion de errores
    //modifico este metodo para que busque  solo id's activos.
```

si se dejan comentarios preferiblemente en inglés por el codigo que está en inglés